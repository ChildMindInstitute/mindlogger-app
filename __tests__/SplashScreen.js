import 'react-native';
import React from 'react';
import SplashScreen from '../app/components/splashscreen';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

 it('renders correctly', () => {
   const tree = renderer.create(
     <SplashScreen />
   );
   expect(tree).toMatchSnapshot();
 });
