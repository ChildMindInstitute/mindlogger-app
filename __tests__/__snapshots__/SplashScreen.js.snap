// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Image
  source={1}
  style={
    Object {
      "flex": 1,
      "height": null,
      "width": null,
    }
  }
/>
`;
